src/EnergyPlus/HeatBalanceSurfaceManager.cc

```diff
@@ -2338,34 +2321,36 @@ void InitThermalAndFluxHistories(EnergyPlusData &state)
         state.dataSurfaceGeometry->kivaManager.surfaceConvMap[SurfNum].out = KIVA_CONST_CONV(0.0);
     }
     if (!state.dataHeatBal->SimpleCTFOnly || state.dataGlobal->AnyEnergyManagementSystemInModel) {
-        for (SurfNum = 1; SurfNum <= state.dataSurface->TotSurfaces; ++SurfNum) {
-            if (!Surface(SurfNum).HeatTransSurf) continue;
-            if ((Surface(SurfNum).ExtBoundCond == ExternalEnvironment) || (Surface(SurfNum).ExtBoundCond == OtherSideCondModeledExt)) {
+        for (int SurfNum = 1; SurfNum <= state.dataSurface->TotSurfaces; ++SurfNum) {
+            auto &surface = state.dataSurface->Surface(SurfNum);
+            if (!surface.HeatTransSurf) continue;
```

Comment:
Use list of heat transfer surfaces.

Reply:
`if (!surface.HeatTransSurf) continue` shows up in 4 other places in this file.